Resources/Audio/_RMC14/Voice/Human/attributions.yml

```diff
@@ -82,3 +82,24 @@
   license: "CC-BY-SA-3.0"
   copyright: "https://github.com/cmss13-devs/cmss13/commit/2b6f77936d1e4b114832c12598ac42947bdfc1a6"
   source: "https://github.com/cmss13-devs/cmss13/blob/92c1793b233955ef8bcd0eaee139736d340c374e/sound/voice/human_male_scream_5.ogg"
+
+- files:
+  - female_preburst1.ogg
+  - female_preburst2.ogg
+  - female_preburst3.ogg
+  - female_preburst4.ogg
+  - female_preburst5.ogg
+  - female_preburst6.ogg
+  - female_preburst7.ogg
+  - male_preburst1.ogg
+  - male_preburst2.ogg
+  - male_preburst3.ogg
+  - male_preburst4.ogg
+  - male_preburst5.ogg
+  - male_preburst6.ogg
+  - male_preburst7.ogg
+  - male_preburst8.ogg
+  - male_preburst9.ogg
+  license: "CC-BY-SA-3.0"
```

Comment:
I checked 13's PR and they are from TGMC which would make these NC
It's not clear if you can use NC assets in a game with a Patreon even if its a free to play game, so we wouldn't be able to use these